### ISSUE TITLE
Accessibility Extension reports an error in the console and aria-label was added incorrectly

### ISSUE THREAD

**Issue by leon0leon** [fusioncharts/fusioncharts-dist#48]
The accessibility extension reported the following error on the console:

![image](https://user-images.githubusercontent.com/123917284/215412471-10f4b975-667c-40d4-a137-a80973915e48.png)

This occurs when the data length of the later series in a scatter chart is less than the data length of the first series.
I noticed that this problem was fixed in @fusioncharts\accessibility\src\categories\multiSeries.js (as shown on the left in the screenshot below), but not in [Accessibility Extension](https://www.fusioncharts.com/extensions/accessibility/) (as shown on the right in the screenshot below):
![image](https://user-images.githubusercontent.com/123917284/215412804-13168cd7-fb59-4d2d-98e3-509549524e12.png)

This also causes the aria-label to be added incorrectly, causing incorrect text to be announced by screen readers. 
Such as in the screenshot below, the data length of the first series (red dots) is 3. Although the data length of the second series (black dots) is greater than 3, it is also incorrectly considered that there are only 3 plots, and only the first 3 points have been added aria-label:
![image](https://user-images.githubusercontent.com/123917284/215434893-80071a31-566a-462f-8a3d-2c97f17a0cad.png)

Another example is the screenshot below. Although the third series (blue) has only one point, which is less than the 35 points of the first series (red), its aria-lable is also added with "Plot 1 of 35". The sentence in the black background in the screenshot below is the text announced by VoiceOver when the data plot in the red circle gets the focus.
![image](https://user-images.githubusercontent.com/123917284/215435177-a10e5ea8-1ab5-4d10-8350-9a603658bc99.png)

**Comment by leon0leon** [comment]
Please note that this problem needs to be reproduced with a scatter type chart. 
The following example can reproduce the problem: https://codesandbox.io/s/accessibility-example-angular-forked-v2h0x4
